### PR TITLE
Alibaba Cloud for ImageNet resnet26d inference on ecs.gn6i-c8g1.2xlarge

### PR DESCRIPTION
**Pull request by lvniqi**
Hello~ we updated the model and framework, and we got the new records of inference time and cost.

according to the [alibabacloud website](https://cn.aliyun.com/price/product?spm=5176.8789780.1092586.1.2d0257a8rKsPw1#/ecs/detail) ,the ecs.gn6i-c8g1.2xlarge is 12.6 RMB per hour.
the US dollar/RMB exchange rate is 0.14, so the cost is 12.6\*0.14/(1000*3600)*0.637316=3.1228484e-07

### CODE REVIEW

File: ImageNet/inference/AlibabaCloud_resnet26d_1t4_ecs_pytorch_tensorRT.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "codeURL": "https://github.com/ali-perseus/DAWNBench_Inference",
+    "commitHash": "bc904f374c4e6335e964ffeb9a96e5a335471f3b",
+    "model": "ResNet26d",
+    "hardware": "Alibaba Cloud [ecs.gn6i-c8g1.2xlarge]",
+    "latency": 0.637316,
+    "cost": 3.1228484e-07,
+    "top5Accuracy": 93.028,
+    "timestamp": "2019-10-17",
+    "misc": {
+        "TensorRT": "TensorRT 6.0.1.5 for CentOS/RedHat 7 (https://developer.nvidia.com/compute/machine-learning/tensorrt/secure/6.0/GA_6.0.1.5/tars/TensorRT-6.0.1.5.CentOS-7.6.x86_64-gnu.cuda-10.0.cudnn7.6.tar.gz",
```
Comment (codyaustun) [inline ImageNet/inference/AlibabaCloud_resnet26d_1t4_ecs_pytorch_tensorRT.json:15]: Please add the version of Pytorch.

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good to me, but 1 small request.

File: ImageNet/inference/AlibabaCloud_resnet26d_1t4_ecs_pytorch_tensorRT.json (hunk 1 above)
Comment (lvniqi) [reply ImageNet/inference/AlibabaCloud_resnet26d_1t4_ecs_pytorch_tensorRT.json]: thanks for reply~ 
we update the Pytorch version and torchvision version now!

Comment (lvniqi) [reply ImageNet/inference/AlibabaCloud_resnet26d_1t4_ecs_pytorch_tensorRT.json]: > Please add the version of Pytorch.

thanks for reply~
we update the Pytorch version and torchvision version now!

Comment (lvniqi) [reply ImageNet/inference/AlibabaCloud_resnet26d_1t4_ecs_pytorch_tensorRT.json]: > Please add the version of Pytorch.

we have updated the Pytorch version and torchvision version in new commit!

Comment (lvniqi) [reply ImageNet/inference/AlibabaCloud_resnet26d_1t4_ecs_pytorch_tensorRT.json:15]: > Please add the version of Pytorch.

Thanks for your reply~ We have updated the Pytorch version and torchvision version in a new commit!

**Review by lvniqi — COMMENTED** [review]
we have updated the Pytorch version and torchvision version in a new commit.

**Comment by FullStackHan** [comment]
Hi, @lvniqi 
I noticed that the latency you reported does not include H2D time (data transferred from the host to the GPU), including only GPU inference + D2H time. Since the inference latency is already below 1ms on T4, the H2D time has become important for latency measurement. 
I am not sure if there are clear rules about whether H2D should be included. @codyaustun

**Comment by lvniqi** [comment]
> Hi, @lvniqi
> I noticed that the latency you reported does not include H2D time (data transferred from the host to the GPU), including only GPU inference + D2H time. Since the inference latency is already below 1ms on T4, the H2D time has become important for latency measurement.
> I am not sure if there are clear rules about whether H2D should be included. @codyaustun

@FullStackHan Thank you for your interest in our work~
In #104, the author has already used [preprocess with GPU](https://github.com/GammaLab-HPC/dawnbench_inference_imagenet/blob/master/test.cpp#L91), and this part is not included in the latency. We just followed this kind of procedure. 
And I think transferring data from host to device is a part of preprocessing step, according to #72 , we can do this offline.

**Comment by codyaustun** [comment]
@FullStackHan I think you bring up a good point, but we have to keep the rules consistent until we update the benchmark. Also, keep an eye on [MLPerf](https://mlperf.org/), which we have been heavily involved in. The first round of inference results should be released in the next few weeks.

**Comment by codyaustun** [comment]
@lvniqi Looks good to me. Thanks!

**Comment by FullStackHan** [comment]
@lvniqi @codyaustun Thank you for your reply to my question.  
your reply is good to me and helped me to eliminate doubts. 👍 
I am also very interested in MLPerf. I will follow the information about MLPerf and look forward to the release of MLPerf inference results. 
Thanks!

**Pull request merged**